src/modules/AddQR/Module.js

```diff
@@ -0,0 +1,50 @@
+module.exports = function AddQR(options, UI) {
+
+    options.size = options.size || 200;
+    options.qrCodeString = options.qrCodeString || "https://github.com/publiclab/image-sequencer";
```

Comment:
Maybe this could link to sequencer.publiclab.org(not necessary) or maybe the README

Reply:
Sure I would do that.